activestorage/lib/active_storage/validations/size_validator.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module ActiveStorage
+  module Validations
+    class SizeValidator < ActiveModel::EachValidator
+      delegate :number_to_human_size, to: ActiveSupport::NumberHelper
+
+      AVAILABLE_CHECKS = [:less_than, :less_than_or_equal_to, :greater_than, :greater_than_or_equal_to, :between]
```

Comment:
From @connorshea's comments below, this could be greatly simplified to 

- `:minimum`
- `:maximum`
- `:in`